src/components/DatePicker/hooks.ts

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from 'react'
```

Comment:
I think you can move this file into the hooks folder as hook

Reply:
I don't think because we are using datepicker localize here wich is mean that this is not a reusable for all components